docs/process_updating_mce_hypershift.md

```diff
@@ -0,0 +1,35 @@
+# Updating Hypershift
+
+The ACM is packaged as an OCM Addon and deploy to the service cluster with fleet manager.
```

Comment:
The ACM is packaged as an OCM Addon and deploy**ed** to the service cluster with fleet manager.